pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "gatorgrade"
+version = "0.1.0"
+description = "Python tool to execute GatorGrader"
+authors = ["Michael Abraham", "Jacob Allebach", "Liam Black", "Katherine Burgess", "Yanqiao Chen", "Ochirsaikhan Davaajambal", "Tuguldurnemekh Gantulga", "Anthony Grant-Cook", "Dylan Holland", "Peyton Kelly", "Luke Lacaria", "Lauren Nevill", "Jack Turner", "Daniel Ullrich", "Garrison Vanzin", "Rian Watson"]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4.0"
+typer = "^0.4.1"
+
+[tool.poetry.dev-dependencies]
+taskipy = "^1.10.1"
+pytest = "^7.1.1"
+pytest-cov = "^3.0.0"
+pylint = "^2.13.4"
+black = "^22.3.0"
+flake8 = "^4.0.1"
+mypy = "^0.942"
+pydocstyle = "^6.1.1"
```

Comment:
Is there a specific motivation for having these additional linters? I wonder if it may be confusing for others since only `pylint` is required for this project.

Reply:
As a group we discussed how adding these additional linters will eventually lead to a cleaner and higher quality final product. We also discussed how some of these enforce a higher level of documentation which could help in the future when thinking about the maintainability and/or sustainability of the system. We did address how adding these extra linters might be more tedious but we overall decided we think their benefits would outweigh the cons.

Reply:
I'm in favor of including all the other linters. I think most people are familiar with them and have used them before in previous classes. I also think that they can set important requirements for the formatting of the project. Pydocsryle can be especially helpful since there are efforts to generate a website for the tool which requires docstrings to be included. However, I'm interested to hear about what others think about including all these linters.